### ISSUE TITLE
why not add hover text for pv/uv?

### ISSUE THREAD

**Issue by xu-song** [theme-next/hexo-theme-next#108]
### I agree and want to create new issue 

- [x] Yes, I was on [Hexo Docs page](https://hexo.io/docs/), especially on [Templates](https://hexo.io/docs/templates.html), [Variables](https://hexo.io/docs/variables.html), [Helpers](https://hexo.io/docs/helpers.html) and [Troubleshooting](https://hexo.io/docs/troubleshooting.html).
- [x] Yes, I was on [NexT Documentation Site](http://theme-next.org/docs/).
- [x] And yes, I already searched for current [issues](https://github.com/theme-next/hexo-theme-next/issues?utf8=%E2%9C%93&q=is%3Aissue) and this is not help to me.

***

### Expected behavior 

hover text for PV&UV， instead of header&footer.

Hover text is more concise than footer & header.

- [demo with hover text](http://xusong.vip/2018/02/09/demo/)   see *pv uv*

pls goto changes https://github.com/theme-next/hexo-theme-next/pull/110/commits/a674f359df2ab44ab79237ea738076a63c8dd2e3

### Actual behavior 

only header & footer is availabel.
- [demo without hover text](https://almostover.ru/)  see *pv uv* & other *count*

### Steps to reproduce the behavior 
1. N/A
2. N/A
3. N/A

* Link to demo site with this issue:      
http://xusong.vip/2018/02/09/demo/

* Link(s) to source code or any usefull link(s): https://github.com/theme-next/hexo-theme-next/pull/110/commits/a674f359df2ab44ab79237ea738076a63c8dd2e3

**Comment by sli1989** [comment]
add or replace in site_uv_header and site_pv_header?

**Comment by xu-song** [comment]
I have take a pull request. The changes can be seen https://github.com/theme-next/hexo-theme-next/pull/110/commits/a674f359df2ab44ab79237ea738076a63c8dd2e3

**Comment by sli1989** [comment]
i mean you can just add sth in `site_uv_header` and `site_pv_header` like `site_uv_header: <i class="fa fa-user"></i> 本站来访数` .

**Comment by xu-song** [comment]
No, they are different.  @sli1989

**Comment by sli1989** [comment]
yeah, thanks, got it.

**Comment by ivan-nginx** [comment]
@xu-song @sli1989 i don't use this service and not totally understand what is this, but i think all titles must at least have transalted languages in similar directory of NexT theme.
NexT is multilanguage theme. So, i think non-Chinese people will not pretty good to see in default configuration Chinese (or any other) languages. U will be glad if i'll add in default config something like:
```yml
new_cool_feature:
  enabled: true
  title: Привет! Вся конфигурация вроде-бы на Английском, но вот именно эта опция такие на Русском. А что такого то?
```
So, please, add `i18n` support.

**Comment by wafer-li** [comment]
Solved by #129
